packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/DroppableItemsList.tsx

```diff
@@ -75,15 +74,6 @@ const DroppableItemsList: FC<DroppableItemsListProps> = ({
                                         mb="xs"
                                         ref={innerRef}
                                         {...draggableProps}
-                                        style={{
```

Comment:
The reason the items were hidden was to hide the problem where the items can't be reordered in the `rows` section.

https://github.com/lightdash/lightdash/assets/9117144/eb18a520-0f94-4fcb-9bbb-6c6ed75b77de

Ideas:
- allow users to reorder the rows ? this might be require a bit of work since we don't support that atm
- only allow the item to be dropped in the correct position
- keep it like this. maybe have a info box explaining why it happens?

cc @magnew

Reply:
It would be great to add row-reordering, but I think we should treat that as a future improvement. How about this:

- Keep both dropzones the way they are, but freeze the elements that aren't being dragged. When an item gets dropped it doesn't change the order, just moves it to the other category. Does that sound like it'll work?

Reply:
That makes sense. 
@Ankush263 I'm going to add all these comments to the main ticket and close this PR but feel free to reopen if you find a solution. Looks like this thread is about the same problem and there are potential solutions there.
https://github.com/atlassian/react-beautiful-dnd/issues/374